tests/console/podman.pm

```diff
@@ -39,7 +39,9 @@ sub run {
     }
 
     if (!check_var('DISTRI', 'microos')) {
-        zypper_call 'in podman';
+        my @pkgs = qw(podman);
+        is_jeos && push(@pkgs, 'podman-cni-config');
```

Comment:
This looks like shell, why not `push(@pkgs, 'podman-cni-config') if is_jeos`?

Reply:
Is there an issue with short circuit?

Reply:
Technically not, but it's not really easy to read and not used anywhere else AFAIK. Though being hard to read is the definition of proper perl coding style...

Reply:
Ok, valid point.

Reply:
Fixed
http://kepler.suse.cz/tests/455#step/podman/9